### PR TITLE
chore: change use case of Endtimestamp to also have starttimestamp

### PR DESCRIPTION
**Pull request by darkvillager2**
## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

File: websites/A/ARD Mediathek/presence.ts [hunk 1]
```diff
@@ -85,10 +85,11 @@ presence.on("UpdateData", async () => {
...
 			)}`;
 
-			[, presenceData.endTimestamp] = presence.getTimestamps(
-				Math.floor(video.currentTime),
-				Math.floor(video.duration)
-			);
+			[presenceData.startTimestamp, presenceData.endTimestamp] =
+				presence.getTimestamps(
+					Math.floor(video.currentTime),
+					Math.floor(video.duration)
```
Comment (Bas950) [inline websites/A/ARD Mediathek/presence.ts:91]: use getTimestampsfromMedia

File: websites/A/AnimeYuzu/presence.ts [hunk 2]
```diff
@@ -217,10 +218,11 @@ presence.on("UpdateData", async () => {
...
 				if (!video.paused) {
-					[, presenceData.endTimestamp] = presence.getTimestamps(
-						Math.floor(video.current),
-						Math.floor(video.duration)
-					);
+					[presenceData.startTimestamp, presenceData.endTimestamp] =
+						presence.getTimestamps(
+							Math.floor(video.current),
+							Math.floor(video.duration)
+						);
```
Comment (Bas950) [inline websites/A/AnimeYuzu/presence.ts:225]: ditto

File: websites/A/AnimixPlay/metadata.json [hunk 3]
```diff
@@ -14,7 +14,7 @@
 		"www1.animixplayer.top",
 		"animixplay.st"
 	],
-	"version": "1.0.2",
+	"version": "1.0.7",
```
Comment (Bas950) [inline websites/A/AnimixPlay/metadata.json:17]: huge version bump?

File: websites/A/AnimixPlay/presence.ts [hunk 4]
```diff
@@ -30,15 +30,13 @@ presence.on("UpdateData", async () => {
 	presenceData.startTimestamp = browsingTimestamp;
 
 	if (href.startsWith("https://www1.animixplayer.top/")) {
-		const [startTimestamp, endTimestamp] = presence.getTimestamps(
+		const titleAndEpisode = document.querySelector(".animetitle").textContent;
+
+		[presenceData.startTimestamp, presenceData.endTimestamp] =
+			presence.getTimestamps(
 				Math.floor(video.currentTime),
 				Math.floor(video.duration)
```
Comment (Bas950) [inline websites/A/AnimixPlay/presence.ts:38]: ditto

File: websites/A/aha/presence.ts [hunk 5]
```diff
@@ -15,12 +15,11 @@ const getVideoStatus = (
...
-				Math.floor(video.currentTime),
-				Math.floor(video.duration)
-			);
-			presenceData.startTimestamp = startTimestamp;
-			presenceData.endTimestamp = endTimestamp;
+			[presenceData.startTimestamp, presenceData.endTimestamp] =
+				presence.getTimestamps(
+					Math.floor(video.currentTime),
+					Math.floor(video.duration)
+				);
```
Comment (Bas950) [inline websites/A/aha/presence.ts:22]: ditto

File: websites/B/Behance/presence.ts [hunk 6]
```diff
@@ -133,10 +133,11 @@ presence.on("UpdateData", async () => {
...
 			if (!video.paused) {
-				presenceData.endTimestamp =
-					Date.now() / 1000 +
-					presence.timestampFromFormat(video.duration) -
-					presence.timestampFromFormat(video.currentTime);
+				[presenceData.startTimestamp, presenceData.endTimestamp] =
+					presence.getTimestamps(
+						Math.floor(video.current),
+						Math.floor(video.duration)
+					);
```
Comment (Bas950) [inline websites/B/Behance/presence.ts:140]: Ditto

File: websites/B/BetterAnime/presence.ts [hunk 7]
```diff
@@ -137,10 +137,8 @@ presence.on("UpdateData", async () => {
 					? (await strings).pause
 					: (await strings).play;
 
-				data.presenceData.endTimestamp = presence.getTimestamps(
-					video.currentTime,
-					video.duration
-				)[1];
+				[data.presenceData.startTimestamp, data.presenceData.endTimestamp] =
+					presence.getTimestamps(video.currentTime, video.duration);
```
Comment (Bas950) [inline websites/B/BetterAnime/presence.ts:141]: ditto

File: websites/B/BetterAnime/presence.ts [hunk 8]
```diff
@@ -204,10 +202,8 @@ presence.on("UpdateData", async () => {
...
```
Comment (Bas950) [inline websites/B/BetterAnime/presence.ts:206]: ditto

File: websites/0-9/2048Verse/presence.ts [hunk 9]
```diff
@@ -67,10 +67,8 @@ presence.on("UpdateData", async () => {
 		} Best ${document.querySelector(".best-container")?.textContent}`;
 	}
 
-	if (!time) {
-		delete presenceData.startTimestamp;
-		delete presenceData.endTimestamp;
-	}
+	if (!time) delete presenceData.startTimestamp;
+
```
[the remark this answers is not in the document]
Comment (darkvillager2) [reply websites/0-9/2048Verse/presence.ts:71]: To meet current guideliness while i was at it for the other thing.

File: websites/A/ARD Mediathek/presence.ts (hunk 1 above)
Comment (darkvillager2) [reply websites/A/ARD Mediathek/presence.ts:91]: here it is required.

File: websites/A/Anime4up/presence.ts [hunk 10]
```diff
@@ -45,10 +45,11 @@ const presence = new Presence({
...
 				run: () => {
-					[, presenceData.endTimestamp] = presence.getTimestamps(
-						Math.floor(video.currentTime),
-						Math.floor(video.duration)
-					);
+					[presenceData.startTimestamp, presenceData.endTimestamp] =
+						presence.getTimestamps(
+							Math.floor(video.currentTime),
+							Math.floor(video.duration)
+						);
```
[the remark this answers is not in the document]
Comment (darkvillager2) [reply websites/A/Anime4up/presence.ts:52]: cant.

File: websites/A/AnimeYuzu/presence.ts [hunk 11]
```diff
@@ -157,10 +157,11 @@ presence.on("UpdateData", async () => {
...
```
[the remark this answers is not in the document]
Comment (darkvillager2) [reply websites/A/AnimeYuzu/presence.ts:164]: cant

File: websites/A/AnimeYuzu/presence.ts (hunk 2 above)
Comment (darkvillager2) [reply websites/A/AnimeYuzu/presence.ts:225]: cant

File: websites/A/AnimesOnline/presence.ts [hunk 12]
```diff
@@ -95,10 +95,11 @@ presence.on("UpdateData", async () => {
...
 		if (!video.paused) {
-			[, presenceData.endTimestamp] = presence.getTimestamps(
-				Math.floor(video.current),
-				Math.floor(video.duration)
-			);
+			[presenceData.startTimestamp, presenceData.endTimestamp] =
+				presence.getTimestamps(
+					Math.floor(video.current),
+					Math.floor(video.duration)
+				);
```
[the remark this answers is not in the document]
Comment (darkvillager2) [reply websites/A/AnimesOnline/presence.ts:102]: cant

File: websites/A/Animevietsub/presence.ts [hunk 13]
```diff
@@ -149,10 +149,11 @@ presence.on("UpdateData", async () => {
...
 			presenceData.smallImageText = video.paused ? strings.pause : strings.play;
-			presenceData.endTimestamp = presence.getTimestamps(
-				Math.floor(video.currentTime),
-				Math.floor(video.duration)
-			)[1];
+			[presenceData.startTimestamp, presenceData.endTimestamp] =
+				presence.getTimestamps(
+					Math.floor(video.currentTime),
+					Math.floor(video.duration)
+				);
```
[the remark this answers is not in the document]
Comment (darkvillager2) [reply websites/A/Animevietsub/presence.ts:156]: here it is required.

File: websites/A/AnimixPlay/metadata.json (hunk 3 above)
Comment (darkvillager2) [reply websites/A/AnimixPlay/metadata.json:17]: i will look into it

File: websites/A/AnimixPlay/presence.ts (hunk 4 above)
Comment (darkvillager2) [reply websites/A/AnimixPlay/presence.ts:38]: cant

File: websites/A/aha/presence.ts (hunk 5 above)
Comment (darkvillager2) [reply websites/A/aha/presence.ts:22]: here it is required.

File: websites/B/Behance/presence.ts (hunk 6 above)
Comment (darkvillager2) [reply websites/B/Behance/presence.ts:140]: cant

File: websites/B/BetterAnime/presence.ts (hunk 8 above)
Comment (darkvillager2) [reply websites/B/BetterAnime/presence.ts:206]: cant

File: websites/B/BetterAnime/presence.ts (hunk 7 above)
Comment (darkvillager2) [reply websites/B/BetterAnime/presence.ts:141]: cant

File: websites/M/MyOnlineRadio/handleStation.ts [hunk 14]
```diff
@@ -75,8 +75,13 @@ export function handleStation(
 						)?.textContent;
 
 					if (startTS) presenceData.startTimestamp = tDiffInMS(startTS);
-					if (startTS && endTS)
-						presenceData.endTimestamp = tDiffInMS(startTS, endTS);
+					if (startTS && endTS) {
```
Comment (Bas950) [inline websites/M/MyOnlineRadio/handleStation.ts:78]: No version bump?

File: websites/M/movie-web/metadata.json [hunk 15]
```diff
@@ -44,7 +44,7 @@
 		"*://movie-web.x/*",
 		"*://sudo-flix.rip/*"
 	],
-	"version": "1.1.4",
+	"version": "1.1.8",
```
Comment (Bas950) [inline websites/M/movie-web/metadata.json:47]: Big increase

File: websites/S/Skribbl.io/metadata.json [hunk 16]
```diff
@@ -21,7 +21,7 @@
 		"nl": "Skribbl.io is een gratis multiplayer teken- en raadspel. Teken en raad woorden met je vrienden en mensen van over de hele wereld! Scoor de meeste punten en wees de winnaar!"
 	},
 	"url": "skribbl.io",
-	"version": "1.2.7",
+	"version": "1.2.8",
```
Comment (Bas950) [inline websites/S/Skribbl.io/metadata.json:24]: Bump but no changes

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/B/BeatLeader/metadata.json [hunk 17]
```diff
@@ -21,7 +21,7 @@
 		"*://*.beatleader.xyz/*",
 		"*://*.beatleader.net/*"
 	],
-	"version": "1.10.1",
+	"version": "1.10.6",
```
Comment (theusaf) [inline websites/B/BeatLeader/metadata.json:24]:
```suggestion
	"version": "1.10.2",
```

File: websites/B/Behance/presence.ts [hunk 18]
```diff
@@ -133,10 +133,11 @@ presence.on("UpdateData", async () => {
...
 			delete presenceData.startTimestamp;
 			if (!video.paused) {
-				presenceData.endTimestamp =
-					Date.now() / 1000 +
-					presence.timestampFromFormat(video.duration) -
-					presence.timestampFromFormat(video.currentTime);
+				[presenceData.startTimestamp, presenceData.endTimestamp] =
+					presence.getTimestamps(
+						Math.floor(video.current),
+						Math.floor(video.duration)
```
Comment (theusaf) [inline websites/B/Behance/presence.ts:139]: Video time stamps are retrieved as text from html elements. Review interface and data when changing this presence.

File: websites/C/Canvas/presence.ts [hunk 19]
```diff
@@ -704,9 +704,8 @@ presence.on("UpdateData", async () => {
 												timeElapsed.getTime() + daySeconds * 1000
 											);
 											delete presenceData.startTimestamp;
-											presenceData.endTimestamp = Math.floor(
-												timeElapsed.getTime() / 1000
-											);
+											[presenceData.startTimestamp, presenceData.endTimestamp] =
+												presence.getTimestamps(elapsed, duration);
```
Comment (theusaf) [inline websites/C/Canvas/presence.ts:708]: Due to the way this presence works, this should probably just set the start timestamp to `Date.now() / 1000` in addition to the previous `endTimestamp` code

File: websites/C/Crave/presence.ts [hunk 20]
```diff
@@ -14,19 +14,18 @@ presence.on("UpdateData", async () => {
...
+					)
 				);
 
-			presenceData.startTimestamp = presence.getTimestamps(
-				elapsed,
-				duration
-			)[0];
-			presenceData.endTimestamp = presence.getTimestamps(elapsed, duration)[1];
+			[presenceData.startTimestamp, presenceData.endTimestamp] =
+				presence.getTimestamps(elapsed, duration);
```
Comment (theusaf) [inline websites/C/Crave/presence.ts:28]: Duplicate fix attempt?
```suggestion
```

File: websites/C/Cuevana 3/presence.ts [hunk 21]
```diff
@@ -126,7 +125,10 @@ presence.on("UpdateData", async () => {
 				presenceData.smallImageText = videoPaused
 					? strings.pause
 					: strings.play;
-				if (!videoPaused) presenceData.endTimestamp = endTimestamp;
+				if (!videoPaused) {
+					[presenceData.startTimestamp, presenceData.endTimestamp] =
+						presence.getTimestampsfromMedia(video);
```
Comment (theusaf) [inline websites/C/Cuevana 3/presence.ts:130]: `video` doesn't exist. Some data is retrieved from an iframe. Consider adding a `startTimestamp` global variable to the original code instead.

File: websites/F/FormulaOneTv/presence.ts [hunk 22]
```diff
@@ -126,7 +115,7 @@ async function setBrowsingActivity(presenceData: PresenceData) {
 	delete presenceData.endTimestamp;
 
 	presenceData.details = "Browsing...";
-	presenceData.startTimestamp = getEpochInSeconds();
+	presenceData.startTimestamp = browingTimestamp;
```
Comment (theusaf) [inline websites/F/FormulaOneTv/presence.ts:118]:
```suggestion
	presenceData.startTimestamp = browsingTimestamp;
```

File: websites/G/Gaia/metadata.json [hunk 23]
```diff
@@ -9,7 +9,7 @@
 		"en": "Gaia offers the largest online resource of consciousness-expanding videos—over 8,000 informative and enlightening films, original shows, yoga and meditation"
 	},
 	"url": "www.gaia.com",
-	"version": "1.0.13",
+	"version": "1.0.15",
```
Comment (theusaf) [inline websites/G/Gaia/metadata.json:12]:
```suggestion
	"version": "1.0.14",
```

File: websites/K/Kayoanime/presence.ts [hunk 24]
```diff
@@ -62,18 +62,14 @@ presence.on("UpdateData", async () => {
...
 						presenceData.smallImageKey = video.paused
 							? Assets.Pause
 							: Assets.Play;
 						presenceData.smallImageText = video.paused
 							? (await strings).pause
 							: (await strings).play;
-						presenceData.startTimestamp = start;
-						presenceData.endTimestamp = end;
+						[presenceData.startTimestamp, presenceData.endTimestamp] =
+							presence.getTimestampsfromMedia(video);
```
Comment (theusaf) [inline websites/K/Kayoanime/presence.ts:72]: `video` is iframe data, not an actual video element.

File: websites/A/AnimixPlay/presence.ts
Comment (theusaf) [inline websites/A/AnimixPlay/presence.ts:1]: Changes made but version not bumped

**Review by theusaf — CHANGES_REQUESTED** [review]

File: websites/S/Skribbl.io/metadata.json (hunk 16 above)
Comment (darkvillager2) [reply websites/S/Skribbl.io/metadata.json:24]: There are changes...

File: websites/Y/YouTube/metadata.json [hunk 25]
```diff
@@ -75,7 +75,7 @@
 		"*://www.youtube.com/*",
 		"*://studio.youtube.com/*"
 	],
-	"version": "5.13.0",
+	"version": "5.13.1",
```
Comment (Bas950) [inline websites/Y/YouTube/metadata.json:78]: bumped without changes for the rest LGTM

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**